src/components/logInForm/LogInForm.js

```diff
@@ -1,19 +1,73 @@
 import styles from "@/styles/logInModal.module.scss";
+import loginUser from "@/utils/loginUser";
 
-const LogInForm = ({setToRegister, setToForgotPassword, setToAnonymous}) => {
+const LogInForm = ({
+  setToRegister,
+  setToForgotPassword,
+  setToAnonymous,
+  setIsLoggedIn,
+  setDisplayPopup,
+  setPopupTitle,
+  setPopupMessage
+}) => {
+  const clickHandler = async (e) => {
+    e.preventDefault();
+    const username = document.querySelector("#user").value;
+    const password = document.querySelector("#password").value;
```

Comment:
You will not see a lot of querySelector in React codebases, for a number of reasons. The recommended way of accessing DOM elements in React is by `useRef`. 
Not saying you have to change it if it works for now, but it's good to know the recommended ways right? 😁

Some reading:
https://www.upbeatcode.com/react/how-to-use-queryselector-in-react/
https://react.dev/learn/manipulating-the-dom-with-refs